packages/babel-plugin-transform-es2015-modules-amd/src/index.js

```diff
@@ -135,6 +135,9 @@ export default function({ types: t }) {
               FACTORY: factory,
             }),
           ];
+
+          // requeue the CallExpression
+          path.requeue(path.get("body.0.expression"));
```

Comment:
Nit: Can we use `#pushContainer` to skip this?

Reply:
Thanks, I changed it and it still works :)